python/pyspark/sql/tests/connect/test_connect_plan_only.py

```diff
@@ -128,6 +128,16 @@ def test_all_the_plans(self):
         self.assertIsNotNone(plan.root, "Root relation must be set")
         self.assertIsNotNone(plan.root.read)
 
+    def test_union(self):
```

Comment:
Actually we should add a comment like:

```python
def test_union(self):
    # SPARK-12345: a short description of the test
    ...
```

per https://spark.apache.org/contributing.html. Let's don't forget this next time :-).